MAUI/Scheduler/accessibility.md

```diff
@@ -0,0 +1,127 @@
+#  Accessibility Support in Maui Scheduler (SfScheduler)
+
+## Keyboard navigation
+
+The `SfScheduler` supports selection using keyboard interactions.
+
+#### Day, Week and WorkWeek views
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves selection to the same time slot on the next day.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves selection to the same time slot on the previous day.<br/><br/></td></tr>
+<tr>
```

Comment:
Mention next or previous date if RTL and LTR

Reply:
Mentioned next or previous date for RTL and LTR as discussed.